templates/fullstack/server/src/schema/generated.graphql

```diff
@@ -11,7 +11,6 @@ type Comment {
   id: ID!
   title: String!
   description: String!
-  note: Note!
```

Comment:
Why don't we have relational fields anymore? @wtrocki

Reply:
We moved to require this to be explicitly in model, but then our community did not like that so we have reverted this and going to restore it in the next release soon